layouts/shortcodes/friend.html

```diff
@@ -0,0 +1,17 @@
+{{ if .IsNamedParams }}
+     {{- $src := .Get "logo" -}}
+     {{- $small := .Get "logo_small" | default $src -}}
+     {{- $large := .Get "logo_large" | default $src -}}
+     <div class="friend-div">
+         <a target="_blank" href={{ .Get "url"  | safeURL }} title={{ .Get "name" }} >
+             <img class="lazyload"
+                  src={{ $src | safeURL }}
+                  data-src={{ $src | safeURL }}
+                  alt={{ .Get "name" }}
+                  data-sizes="auto"
+                  data-srcset="{{ $small | safeURL }}, {{ $src | safeURL }} 1.5x, {{ $large | safeURL }} 2x" />
+             <span class="friend-name">{{ .Get "name" }}</span>
+             <span class="friend-info">{{ .Get "word" }}</span>
```

Comment:
`"` should be added around these attribute values, e.g., `href="{{ .Get "url" | safeURL }}"`.